config/fam_memoryserver_config.yaml

```diff
@@ -1,4 +1,4 @@
 # MemoryServer Config file
 # Path where data is stored.
-# By default NVMM default location will be selected (which by default is /dev/shm/<username>.
-# data_path: /dev/shm/
+# By default NVMM default location will be selected (which by default is /dev/shm/<username>).
+# fam_path: /dev/shm/
```

Comment:
What happens if a user with <username> = "radixtree" exists in the system? Suggest you use
# fam_path: /dev/shm/user/<username> to maintain separation of user data from other data.